pages/pipelines/command_step.md.erb

```diff
@@ -256,6 +256,10 @@ You can also see when a job has been retried and whether it was retried automati
 
 <%= image "hidden-jobs.png", width: 1400, height: 330, alt: "Retry history is preserved and can be viewed" %>
 
+In the Buildkite UI, there is a [Job Retries Report section](https://buildkite.com/organizations/~/reports/job-retries) where you can view a graphic report on jobs retried within the last 24 hours and up to 30 days in the past. This can help you understand flakiness and instability across all of your pipelines. The report also allows you to view top manual and automatic retries.
```

Comment:
> within the last 24 hours and up to 30 days in the past

Is this different from saying "jobs retried within the last 30 days" ? 

Could even go 

> jobs retried manually or automatically within the last 30 days?

Reply:
I wanted to specify the min and max time frames for job retry reports, but I do agree that it's not the most crucial info here. Rewriting.